code/modules/mob/living/carbon/superior_animal/giant_spider/attack.dm

```diff
@@ -1,5 +1,7 @@
 /mob/living/carbon/superior_animal/giant_spider/UnarmedAttack(var/atom/A, var/proximity)
 	. = ..()
+	if(!..())
```

Comment:
Now you're calling the parent `UnarmedAttack` proc twice.
I would recommend replacing the `!..()` by just `!.`, since the result of `..()` that would get inverted by the `!` was already in the `.` variable.

Reply:
Changed and tested. Done.